### DISCUSSION TITLE
WhatsApp Photo and Video Downloader Extension

### DISCUSSION THREAD

**Discussion by v4r1able** [community/community#37885] [Programming Help]
i coded a new plugin. This plugin can download self-destructing photos and videos on WhatsApp Web

I'd appreciate it if you could point out the bugs you found and where I need to improve :)

View: [https://github.com/v4r1able/wa-downloader](https://github.com/v4r1able/wa-downloader)

**Comment by ossanna16** [comment]
Hi @v4r1able, we’re glad you’re here 😀
Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
